public/assets/styles/variables/_text.scss

```diff
@@ -1,4 +1,4 @@
-$font-base: 'Inter', ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", "Helvetica Neue", Arial, "Noto Sans", sans-serif,
+$font-base: 'IBM Plex Sans Arabic', ui-sans-serif, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", "Helvetica Neue", Arial, "Noto Sans", sans-serif,
```

Comment:
We can't change the font, this change would make that font the default for all. We need to work out a solution that will choose Inter for all non-arabic systems, and use your font for all users who those the arabic language?

Reply:
Agree with you, maybe some condition